### PR TITLE
extend dependabot updates schedule interval from weekly to monthly

### PR DESCRIPTION
**Pull request by slimreaper35**
issue: https://github.com/containerbuildsystem/cachi2/issues/450

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] Docs updated (if applicable)
- [ ] Docs links in the code are still valid (if docs were updated)

**Note:** if the contribution is external (not from an organization member), the CI
pipeline will not run automatically. After verifying that the CI is safe to run:

- [approve GitHub Actions workflows][approve-gh-actions] by clicking a button
- approve the Red Hat Trusted App Pipeline container build by commenting `/ok-to-test`
  (as is the [standard for Pipelines as Code][pac-running-pipeline])

[approve-gh-actions]: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks
[pac-running-pipeline]: https://pipelinesascode.com/docs/guide/running/#running-the-pipelinerun

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
+      - slimreaper35
+      - taylormadore
     groups:
-      dep-patch-updates:
-        update-types:
-          - "patch"
-      dep-minor-updates:
+      minor-and-patch:
+        patterns:
+          - "*"
```
Comment (eskultety) [inline .github/dependabot.yml:25]: Isn't `patterns=*` the default (although the docs doesn't say that explicitly)? Also, do we need to specify this field explicitly? The way I understand
> You must use the patterns, exclude-patterns, dependency-type, or update-types options to define the group, or any combination thereof.

is that `update-types` is enough for us to successfully define a group in dependabot config.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,26 +1,40 @@
...
         update-types:
           - "minor"
+          - "patch"
```
Comment (eskultety) [inline .github/dependabot.yml:28]: This unification should be in a separate commit, it's a self-contained change -> different commit.

File: .github/dependabot.yml [hunk 3]
```diff
@@ -13,12 +13,10 @@ updates:
     schedule:
       interval: "monthly"
     groups:
-      dep-patch-updates:
-        update-types:
-          - "patch"
-      dep-minor-updates:
+      minor-and-patch:
```
Comment (eskultety) [inline .github/dependabot.yml:16]: **commit message:**

I'd like to see some reasoning in the commit message, e.g. "Decrease the PR noise from dependabot by merging both 'patch' and 'minor' updates".

File: .github/dependabot.yml [hunk 4]
```diff
@@ -1,17 +1,17 @@
 # Please see the documentation for all configuration options:
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
```
[the remark this answers is not in the document]
Comment (slimreaper35) [reply .github/dependabot.yml:2]: Can multiple commits close the same issue?

[the remark this answers is not in the document]
Comment (slimreaper35) [reply .github/dependabot.yml:2]: I did on both 😄. The comment at the top of the PR template has not been updated.

**Review by eskultety — APPROVED** [review]

**Review by ejegrova — APPROVED** [review]
